src/components/Markdown.js

```diff
@@ -77,20 +77,18 @@ const renderMath = (node) => <Latex trusted={false} displayMode>{`$$ ${node.valu
 
 const renderInlineMath = (node) => <Latex trusted={false}>{`$ ${node.value} $`}</Latex>;
 
-const renderUserlink = ({ name }) => {
-  return (
-    <a href={`/user/view/${name}`} target="_blank" rel="noopener noreferrer">
-      @{name}
-    </a>
-  );
-};
+const renderUserlink = ({ name }) => (
+  <a href={`/user/view/${name}`} target="_blank" rel="noopener noreferrer">
+    @{name}
+  </a>
+);
 
 const renderSymbol = ({ value }) => {
   const symbol = value.replace('/', '-').toLowerCase();
   return <img src={`/content/symbols/${symbol}.png`} alt={symbol} className="mana-symbol-sm" />;
 };
 
-const renderCardlink = ({ name, id, dfc }) => {
+function renderCardlink({ name, id, dfc }) {
```

Comment:
Why was this one changed to a function when all the ones around it stayed as const lambdas?